cmd/ctr/commands/images/images.go

```diff
@@ -22,12 +22,15 @@ var Command = cli.Command{
 	Name:  "images",
 	Usage: "manage images",
 	Subcommands: cli.Commands{
-		listCommand,
 		checkCommand,
+		exportCommand,
+		importCommand,
+		listCommand,
+		pullCommand,
+		pushCommand,
+		pushObjectCommand,
```

Comment:
This should probably go next to `fetch-object`. The object shouldn't need to be part of an image.

Reply:
good catch, i missed that one